lr-cli/cortx_setup/commands/node/finalize.py

```diff
@@ -38,7 +38,7 @@ def _validate_health_map(self):
         self.logger.debug("Validating node health check")
         resource_paths = [HEALTH_PATH, MANIFEST_PATH]
```

Comment:
can you remove MANIFEST_PATH as we don't have manifest in place

Reply:
Okay.